Makefile.am

```diff
@@ -32,6 +32,10 @@ OSX_PLIST=$(top_builddir)/share/qt/Info.plist #not installed
 OSX_QT_TRANSLATIONS = da,de,es,hu,ru,uk,zh_CN,zh_TW
 
 DIST_DOCS = $(wildcard doc/*.md) $(wildcard doc/release-notes/*.md)
+DIST_CONTRIB = $(wildcard $(top_srcdir)/contrib/*.bash-completion) \
+	       $(top_srcdir)/contrib/debian \
```

Comment:
Thanks, yes, the licensing is still annoying.

Would be easy enough to write our own desktop and protocol file and install them, though OTOH I suppose those things are quite distro specific, at least where to install them.

Out of scope for this pull at least.